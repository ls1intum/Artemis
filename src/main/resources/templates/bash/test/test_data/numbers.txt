2.718
2.718 2.718 2.718
21718
