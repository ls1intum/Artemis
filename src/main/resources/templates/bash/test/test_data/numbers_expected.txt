3.1415
3.1415 3.1415 3.1415
21718
